src/ui/public/field_editor/lib/__tests__/copy_field.test.js

```diff
@@ -17,26 +17,22 @@
  * under the License.
  */
 
-import './duration.less';
-import durationTemplate from './duration.html';
+import { copyField } from '../copy_field';
 
-export function durationEditor() {
-  return {
-    formatId: 'duration',
-    template: durationTemplate,
-    controllerAs: 'cntrl',
-    controller() {
-      this.sampleInputs = [
-        -123,
-        1,
-        12,
-        123,
-        658,
-        1988,
-        3857,
-        123292,
-        923528271
-      ];
-    }
-  };
-}
+const field = {
+  name: 'test_field',
+  scripted: true,
+  type: 'number',
+  lang: 'painless',
+};
+
+describe('copyField', () => {
+  it('should copy a field', () => {
```

Comment:
I was expecting to see some assertions about how the contents have been copied over in some way -- do you think it would make sense or be possible to add some?

Reply:
added some additional assertions